session_test.go

```diff
@@ -30,8 +30,8 @@ func TestNew_BadOption(t *testing.T) {
 func TestNewWithRemoteKey(t *testing.T) {
 	// Act.
 	var (
-		si, err = NewWithRemoteKey(sk, bobPair.PublicKey())
-		s       = si.(*session)
+		si, err = NewWithRemoteKey([]byte("id"), sk, bobPair.PublicKey(), nil)
```

Comment:
This `err` is still pointed out by lint (we need to either check it or replace it with `_`).